features.html

```diff
@@ -1,322 +1,475 @@
+<div class="feature-section">
+  <div class="item_toptext">
+    <p>Code blocks inside doc comments are treated as valid Rust code: you get full code insight including syntax highlighting, <a href="#completion">completion</a>, and <a href="#intentions">intentions</a>.
+    </p>
+    <p></p>
+    <img src="/assets/features/smart_editor/doc_comments_hl.png" alt="doc comments highlighting" width="700"/>
   </div>
+</div>
 
-  <div class="feature">
-    <h3>Goto Declaration</h3>
-    <p><kbd>Ctrl+B</kbd> goes to the definition of the symbol at caret.</p>
+<h2 id="editor_formatting" align="left">Code style and formatting</h2>
+<div class="feature-section">
+  <div class="item_toptext">
+    <p>IntelliJ Rust helps your code stay compliant with style guidelines and naming conventions. You can configure the details of the preferred style in <strong>Settings / Preferences | Editor | Code style | Rust</strong>, and adjust the <a href="#inspections">inspections</a> that validate your code against Rust naming conventions.</p>
+    <p>To explicitly reformat your code, you can use the built-in action (<code>Code | Reformat</code> or <code>Ctrl+Alt+L / ⌥⌘L</code>), or call <a href="https://github.com/rust-lang/rustfmt">rustfmt</a> (<code>Code | Reformat Cargo Project with Rustfmt</code> or using the button in the Cargo tool window).</p>
   </div>
+</div>
 
-  <div class="feature">
-    <h3>Goto Class</h3>
-    <p><kbd>Ctrl+N</kbd> search for a struct or enum by name.</p>
-  </div>
 
-  <div class="feature">
-    <h3>Goto Symbol</h3>
-    <p><kbd>Ctrl+Shift+Alt+N</kbd> searches for any symbol (types, methods, functions, fields) by name. Together with
-      <kbd>Ctrl+N</kbd> it is the main method to explore large projects.</p>
+<h2 id="intentions" align="left">Intentions and quick-fixes</h2>
+<div class="feature-section">
+  <div class="item_col1">
+    <!--img class="gif" data-wait="true" data-scope="gif-scope" style="width: auto"
+         src="/assets/features/code_generation/missing_fields.png" data-gif="/assets/features/code_generation/missing_fields.gif"
+         alt="missing fields intention"/-->
+    <img src="/assets/features/code_generation/missing_fields_img.png" alt="missing fields intention"/>
+    <p></p>
+    <p>The <i>Add missing fields</i> quick fix helps you interactively define field values of struct literals.</p>
   </div>
-
-  <div class="feature">
-    <h3>Goto Super</h3>
-    <p><kbd>Ctrl+U</kbd> navigates to the parent module.</p>
+  <div class="item_col2">
+    <img src="/assets/features/code_generation/add_derive_clause_2.png" alt="add derive clause intention"/>
+    <p></p>
+    <p>You can quickly add <code>#[derive]</code> clauses and <code>impl</code> blocks for structs and enums, and perform struct-to-tuple and tuple-to-struct conversion</p>
   </div>
-
-  <div class="feature">
-    <h3>File Structure</h3>
-    <p><kbd>Ctrl+F12</kbd>, <kbd>Alt+7</kbd> brings up an overview of the current file.</p>
+  <div class="item_bottomtext">
```

Comment:
This item looks not uniform with others
![image](https://user-images.githubusercontent.com/2539310/69034883-4bb3d100-09f3-11ea-8f8a-582133925e04.png)